### DISCUSSION TITLE
Stuck in a log-in loop an can't use my paid account to open a ticket... because I can't log-in

### DISCUSSION THREAD

**Discussion by myksensei** [vercel/community#7157] [Account Issue]
### Question

Every time I try to log-in, it takes me to this sms verification page, which then tells me that this account is already associated with the phone number.  So I can't log-in.  So I cannot use my paid service... and cannot submit a ticket...

<img width="544" alt="image" src="https://github.com/vercel/community/assets/160574430/6a2ef362-fa9c-4178-9c41-7d0889d42802">

**Comment by pawlean** [comment]
Hi @myksensei!

Sorry to hear this - did you manage to get in touch with us?

**Reply by myksensei** [reply]
Nope, have given up...

**Comment by amyegan** [comment]
Thanks for reaching out about this. If you send an email to [registration@vercel.com](mailto:registration@vercel.com) from the same email you use on your Vercel account, that will open a new ticket with the team that can help with lost account access.

https://vercel.com/guides/why-can-i-not-signup

[a comment by an automated account is not shown]
